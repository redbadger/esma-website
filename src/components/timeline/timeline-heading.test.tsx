import React from "react";
import renderer from "react-test-renderer";
import TimelineHeading from "./timeline-heading";
import { useStaticQuery } from "gatsby";

describe("TimelineHeading", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<TimelineHeading />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
