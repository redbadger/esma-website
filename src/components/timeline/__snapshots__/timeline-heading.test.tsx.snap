// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TimelineHeading renders correctly 1`] = `
<div
  css={
    Object {
      "map": undefined,
      "name": "nsdht3",
      "next": undefined,
      "styles": "
  text-align: center;
  display: flex;
  flex-direction: column;
  align-items: center;
  h1 {
    color: #2d3142;
    font-size: 2rem;
    line-height: 2.75rem;
    font-weight: 600;
  }

  p {
    color: #2d3142;
    font-size: 1rem;
    line-height: 1.875rem;
    margin-right: 2.5rem;
    margin-left: 2.5rem;
  }

  @media (min-width: 1024px) {
    p {
      width: 56rem;
    }
  }

  a {
    text-decoration: underline;
    font-weight: 600;
  }
",
      "toString": [Function],
    }
  }
>
  <h1>
    A lifetime of social inequality
  </h1>
  <p>
    Weâ€™ve commissioned research from
     
    <a
      href="https://www.thebridgegroup.org.uk"
    >
      The Bridge Group
    </a>
     to build the evidence base needed to make change happen. This research highlights some of the significant ways in which socio-economic background can intersect with other protected characteristics during someones lifetime to contribute to weak social mobility in the UK.
  </p>
</div>
`;
